### PR TITLE
Reverted purchase/renew links back to popup modals with iframe cart

### PR DESCRIPTION
**Pull request by jacobd91**
This PR reverts all changes made to the purchase/renew buttons to restore the popup modals and iframe embeds of the cart. This allows for users to stay on-site and to automatically apply licenses on purchase

### CODE REVIEW

**Review by cssjoe — CHANGES_REQUESTED** [review]
Add escape and translate functions.

File: FeatureShowcase_Plugin_Admin_View.php [hunk 1]
```diff
@@ -75,18 +75,13 @@
...
-			
-		<?php
-	} elseif ( ! empty( $card['button'] ) ) {
-		echo $card['button'];
-	}
-
-	?>
+					<?php
+					if ( $is_premium && ! $is_pro ) {
+						echo '<button class="button w3tc-gopro-button button-buy-plugin" data-src="feature_showcase">Unlock Feature</button>';
```
[the remark this answers is not in the document]
Comment (jacobd91) [reply FeatureShowcase_Plugin_Admin_View.php]: Translation added and switch from echo to php break to print HTML

File: Licensing_Plugin_Admin.php [hunk 2]
```diff
@@ -45,9 +47,9 @@ public function w3tc_admin_bar_menu( $menu_items ) {
 						),
 					)
 				),
-				'href'   => esc_url( 'https://www.boldgrid.com/w3-total-cache/' ),
-				'meta'   => array(
-					'target' => '_blank',
+				'href'   => wp_nonce_url(
```
[the remark this answers is not in the document]
Comment (jacobd91) [reply Licensing_Plugin_Admin.php]: wp_nonce_url escapes the URL

File: Licensing_Plugin_Admin.php [hunk 3]
```diff
@@ -57,10 +59,7 @@ public function w3tc_admin_bar_menu( $menu_items ) {
 				'id'     => 'w3tc_debug_overlay_upgrade',
 				'parent' => 'w3tc_debug_overlays',
 				'title'  => esc_html__( 'Upgrade', 'w3-total-cache' ),
-				'href'   => esc_url( 'https://www.boldgrid.com/w3-total-cache/' ),
-				'meta'   => array(
-					'target' => '_blank',
-				),
+				'href'   => wp_nonce_url( network_admin_url( 'admin.php?page=w3tc_dashboard&amp;w3tc_message_action=licensing_upgrade' ), 'w3tc' ),
```
[the remark this answers is not in the document]
Comment (jacobd91) [reply Licensing_Plugin_Admin.php:59]: wp_nonce_url escapes the URL

**Review by cssjoe — APPROVED** [review]

**Pull request merged**